c/openbsd-6.2/preprocess.pl

```diff
@@ -0,0 +1,11 @@
+if ($_ =~ "#include") {
```

Comment:
Can you please rewrite this in Python or Shell?

Having yet another language for scripts in this repository will complicated maintaining it even more. Especially given that I cannot even understand this source code although it is only a few lines!

Reply:
I now wrote it in awk, which is compliant with `IEEE Std 1003.1-2008 (POSIX.1)` and one of many recommended ways to process text with shell.

I personally don't think the new script is easier to read and I am open for suggestions.

Reply:
It is irrelevant whether this is Posix compliant or not, we need to be able to maintain it. And there will be much more people among the current and future maintainers of this repository who can understand Python or Shell than who can understand Perl or Awk. Furthermore, Python and Shell are already used, but Perl and Awk are not, so using them adds another language as burden on maintainers for little gain.

It is not even documented what this script attempts to do, and variable names like `eschdr` are not helpful. Programs should primarily be written for humans to understand, and only secondarily for computers to execute!